sdk/nodejs/src/cloudClient.ts

```diff
@@ -22,6 +22,14 @@ const castIncomingUser = (user: DVCUser) => {
     return user
 }
 
+const throwIfUserError = (err: AxiosError) => {
+    const data = err.response?.data as { message: string, statusCode: number }
+
+    if (400 <= data.statusCode && data.statusCode < 500) {
```

Comment:
you should read the status code of the response object here, not the data. Data is our response body from the server, which for some reason contains a status code that is the same as the one on the response itself. Just use `response.status` or whatever it is